tests/mk_test.sh

```diff
@@ -144,6 +147,15 @@ function mk_build_Test()
   assertEquals "($ID)" "$expected_result" "${output}"
 
   ID=5
+  declare -a expected_cmd=(
+    "$release"
+    "$modules"
+  )
+  touch "$PWD/modules.order"
+  output=$(mk_build 'TEST_MODE' '--info')
+  compare_command_sequence expected_cmd[@] "$output" "$ID"
+
+  ID=6
```

Comment:
As I suggested in the previous comment, isolate the information in a single function and add proper test coverage.